nameserver/mocks_test.go

```diff
@@ -436,6 +437,7 @@ func (clk *mockedClock) Forward(secs int) {
 	Debug.Printf(">>>>>>> Moving clock forward %d seconds - Time traveling >>>>>>>", secs)
 	clk.Add(time.Duration(secs) * time.Second)
 	Debug.Printf("<<<<<<< Time travel finished! We are at %s <<<<<<<", clk.Now())
+	runtime.Gosched() // give some time to other goroutines
```

Comment:
Is this supposed to be here? If it is it could use a comment explaining why - the current one is redundant